### ISSUE TITLE
Naming convention for typenames is non-configurable and default is not consistent with the default of the typescript plugin

### ISSUE THREAD

**Issue by mdazy** [ardeois/graphql-codegen-typescript-mock-data#16]
The hardcoded behavior for typenames appears to be to retain the casing, whereas the default for the `typescript` plugin is to use `pascalCase`. As a result, there is a mismatch between the types generated by the two plugins: a type name such as `ETPObject` yields `EtpObject` with `typescript`and `ETPObject` with `typescript-mock-data`.

Fortunately the `typescript` plugin allows to change the naming convention for typenames, so this can be worked around like this:
```
generates:
  ./schema/lib/clientTypes.tsx:
    plugins:
      - typescript
    config:
      namingConvention:
        typeNames: keep # for compatibility with typescript-mock-data
```

However, this means I need to adjust my existing code because hooks such as `useGetEtpObjects` have become `useGetETPObjects`. 

It would be good to have matching default behaviors and maybe also matching configuration options.

Thanks.

**Comment by mdazy** [comment]
I'm going to try and propose a PR for this.

**Comment by ardeois** [comment]
Fixed in version 0.4.0
Thanks for the PR @mdazy !
